src/libs/OptionsListUtils.js

```diff
@@ -800,6 +802,9 @@ function getHeaderMessage(hasSelectableOptions, hasUserToInvite, searchValue, ma
         if (/^\d+$/.test(searchValue) && !isValidPhone) {
             return Localize.translate(preferredLocale, 'messages.errorMessageInvalidPhone');
         }
+        if (/@/.test(searchValue) && !isValidEmail) {
+            return Localize.translate(preferredLocale, 'common.error.invalidEmail');
+        }
```

Comment:
Maybe we could add a default, for the case where it's not clearly a phone number or email, but it's invalid for both?

For example when I enter `new1:1` it should say "Invalid email or phone number".

Reply:
1. I think that when user has entered the character as a letter that means they want to search by name or email
2. If you want do add this case, can you defind when `searchValue` is not clearly a phone number or email? When it has both letters and numbers, but doesnt't have '@' ?